src/azure-cli/requirements.py3.windows.txt

```diff
@@ -119,7 +119,7 @@ pbr==5.3.1
 pkginfo==1.8.2
 portalocker==2.3.2
 psutil==5.9.0
-psycopg2==2.9.3
+psycopg2==2.9.5
```

Comment:
IMHO, complex dependencies such as `psycopg2` should be used in an extension, not main Azure CLI.